pallets/loans/src/benchmarking.rs

```diff
@@ -478,8 +478,10 @@ benchmarks! {
 	}: _(RawOrigin::Signed(borrower), pool_id, loan_1, loan_2, repaid_amount, borrow_amount)
 
 	apply_transfer_debt {
+		let n in 2..Helper::<T>::max_active_loans() - 2;
+
 		let any = account("any", 0, 0);
-		let pool_id = Helper::<T>::prepare_benchmark();
+		let pool_id = Helper::<T>::initialize_active_state(n);
```

Comment:
@lemunozm I noticed that the loans benchmarks did parametrize `apply_transfer_debt` which created compiler issues as the `pallet_loans::weights::WeightInfo` expect the existence of a parameter. AFAICT, this provided solution should be appropriate. WDYT?

Reply:
My fault! You did the perfect fix for this! 💯 

Thanks, and sorry for missing this